src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -13,6 +13,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
+#pragma warning disable CA1416 // Validate platform compatibility
```

Comment:
This is the only false positive i got. The file is loaded [when ](https://github.com/dotnet/runtime/blob/6267bec40223b998283185814ad4e9abcf2708dd/src/libraries/System.Net.WebSockets.Client/src/System.Net.WebSockets.Client.csproj#L13-L14)`'$(TargetsBrowser)' != 'true'`. I have tried to exclude this from build with filter: `'($(TargetsAnyOS)' == 'true' and '$(TargetsBrowser)' != 'true')`  but it didn't work and couldn't find other way, i might leave it suppressed as it will not be loaded for browser (There is browser version WebSocketHandle.Browser.cs )

Reply:
CC @safern @ViktorHofer